core/src/test/scala/com/mesosphere/usi/core/SchedulerTest.scala

```diff
@@ -56,7 +56,8 @@ class SchedulerTest extends AkkaUnitTest with Inside {
             Goal.Running,
             RunSpec(
               resourceRequirements = List(ScalarRequirement.cpus(1), ScalarRequirement.memory(256)),
-              shellCommand = "sleep 3600")
+              shellCommand = "sleep 3600",
+              "*")
```

Comment:
You should never launch a task as "*", Mesos will never allocate resources to that role. I know this just a test fixture, can we please change this to an actual role to avoid setting the wrong precedent?

Reply:
so how do you launch podspec on unreserved resources then?

Reply:
You just let the allocator allocate the resources for you

Reply:
(see AllocationInfo)

Reply:
https://github.com/apache/mesos/blob/master/include/mesos/v1/mesos.proto#L1238

Reply:
got it! thanks